uktrade/data-hub-frontend
# Feature/cls2 73 company activity tab add filter for date

## Description of change

On the Company > Activity page allow the user to filter by date.

## Test instructions

Visit a company's activity page, e.g. [Alphabet](http://localhost:3000/companies/9aacdbb0-32b3-4dc5-a961-24e6e6c57a46/activity) or [One List Corp](http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/activity). You should see From and To filters that allow you to limit the date range in the same way as is done on http://localhost:3000/interactions?sortby=date:desc&page=1

## Screenshots

### Before

![image](https://github.com/uktrade/data-hub-frontend/assets/699259/7f23304e-2bb3-46a2-8f03-7ad1f17f20b0)
### After

![image](https://github.com/uktrade/data-hub-frontend/assets/699259/eeee1f65-1299-4f27-962b-4c7be83dd102)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
